tests/test_models.py

```diff
@@ -95,8 +91,8 @@ def list_tuned_models(
                 request = glm.ListTunedModelsRequest(page_size=page_size, page_token=page_token)
             self.assertIsInstance(request, glm.ListTunedModelsRequest)
             self.observed_requests.append(request)
-            response = self.responses["list_tuned_models"][request.page_token]
-            return ListWrapper(response)
+            response = self.responses["list_tuned_models"]
+            return (item for item in response)
```

Comment:
> @add_client_method is probably adding indirection that prevents checking that the fake impl matches the real impl too.

Yeah, this could just be a subclass of `glm.ModelServiceClient`?